### PR TITLE
Make dio calibration of UHFQC compatible with LabOne 22.02

### LINKED ISSUE #687: Make DIO calibration compatible with LabOne 22.02
In LabOne 22.02 the behavior of real-time (RT) logging changed with new public nodes introduced. The PycQED driver implementation of the calibration relied on raw nodes. Adapt the PycQED driver for LabOne 22.02

Acceptance criteria
- The DIO calibration can be performed the usual way also with LabOne 22.02.
- The adapted driver works for older LabOne versions in a way transparent to the user.
- Public nodes are used wherever possible.

### PR DESCRIPTION
**Pull request by martinr-zi**
Fixes fixes #687

@schenfab, @mariosk-zi @michelecollodo

### CODE REVIEW

**Review by mariosk-zi — APPROVED** [review]

**Review by schenfab — CHANGES_REQUESTED** [review]
The nodes are not yet correctly used for the new scheme with >=22.02.

The old usage concept was along this line:
```
self.setd('raw/dios/0/delay', delay)
time.sleep(0.5)
[...]
error_timing = self.geti('raw/dios/0/error/timing')
```

The new concept should look like:
```
self.setd('raw/dios/0/delay', delay)
self.setd('raw/dios/0/error/timingclear', 0xffffffff) # clear the sticky timing error detection bits
time.sleep(0.5)
[...]
error_timing = self.geti('raw/dios/0/error/timingsticky') # readout timing errors that accumulated since the last call to "timingclear"
```

The important improvement here is the addition of the "timingclear" node, which is required to be manually set before checking for timing errors (by sleeping some time and reading the sticky timing error node afterwards). Previously, this clearing was done automatically by the device at arbitrary points in time, meaning reading the timing error node could have lead to false negatives (one would have got no timing errors, while actually there were some that were auto-cleared by the device just before reading the node).

Long story short: You first need to figure out if the old or new nodes are present, and then conditionally also set the "timingclear" node before sleeping... Probably by adding yet another try construct for the "timingclear" (except case would `pass` here), or maybe a bit more nicely, use listNodes beforehand to figure out which scheme is used and get rid of the try/excepts...

**Review by schenfab — APPROVED** [review]
LGTM, thanks for the change!

**Review by wvlothuizen — APPROVED** [review]
Looks good
